File: 2026-08/src/day_24/prms_120824/Solution.java
package day_24.prms_120824;

class Solution {
    public int[] solution(int[] num_list) {
        int odd = 0;
        for (int x : num_list) {
            if (x % 2 != 0) odd++;
        }

        return new int[]{num_list.length - odd, odd};
    }
}

File: 2026-08/src/day_24/prms_120824/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(vector<int> num_list) {
    int odd = 0;
    for (int x : num_list) {
        if (x % 2) odd++;
    }

    return {(int)num_list.size() - odd, odd};
}
